blog/2024/09-10-release-roadmap-10.10.0/index.mdx

```diff
@@ -0,0 +1,81 @@
+
+### The Release Timeline
+
+- Monday, September 9th, 2024 (unstable `20240909`): Feature PR last-call, first beta. Any *new* feature PRs should get in right now to avoid missing the feature freeze. This is your 1 week warning! Developers/contributors **please keep on top of your PR review feedback** or your PR may **miss the following deadlines and have to wait** until 10.11.0.
+- Monday, September 16th, 2024 (unstable `20240916`): Feature freeze, second beta. Any *new* feature PRs after this point must wait until 10.11.0 (in another ~6 months). In-flight feature PRs will likely make it assuming that all feedback is resolved before the API freezes. Again **developers/contributors please keep on top of your PR review feedback**. New PRs should be **bugfixes only** after this point. Users **should be safe to begin testing in earnest** with this unstable.
+- Monday, September 23rd, 2024 (unstable `20240923`): API freeze, third beta. **All feature PRs should be merged by this point**. If they're not, they will be moved to the 10.11.0 project and only merged to `master` once 10.10.0 is released. API is now soft-frozen so that **client developers can begin confidently working on any client support changes needed for 10.10.0**.
+- Monday, September 30th, 2024 (unstable `20240930`): API lock, first release candidate. **Major bugfix PRs should hopefully be either be merged or well in-flight** by this point. Only **non-API-changing bugfixes** after this point. We should have a feature overview blog post in draft state by this point.
+- Monday, October 7th, 2024 (unstable `20241007`): Final planned unstable, second release candidate. **The release should basically be ready to go**, and any remaining bugfix PRs **should be merged before this unstable**.
+- Saturday, October 12th, 2024: Release of 10.10.0!
+
+### Possible Timeline Changes
+
+Since this is a much smaller and quicker release than 10.8.0 or 10.9.0 were, hopefully this is enough time to get everything sorted and ready to go. But if we find it's **not**, we reserve the right to add an extra week in one or both of the following places:
+
+- After the Feature freeze, if we find that there are any outstanding **major** bugfixes that require additional time to merge before the API freeze.
+- After the API lock, if we find that there are any outstanding **non-API-breaking** bugfixes that require additional time to merge before the final unstable.
+
+If neither of those happen, the release will proceed as planned; otherwise, an extra step will be added into the timeline after the given points that matches the previous stage. This might push the final unstable back to either October 14th or October 21st, and the final release to either October 19th or October 26th. If this is relevant to your interests please keep an eye on our [Announcements Matrix channel](https://matrix.to/#/#jellyfin-announce:matrix.org) for announcements.
```

Comment:
```suggestion
If neither of those happen, the release will proceed as planned; otherwise, an extra step will be added into the timeline after the given points that matches the previous stage. This might push the final unstable back to either October 14th or October 21st, and the final release to either October 19th or October 26th. If this is relevant to your interests, please keep an eye on our [Announcements Matrix channel](https://matrix.to/#/#jellyfin-announce:matrix.org) for announcements.
```